client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -81,7 +81,9 @@ class DomainRegistrationSuggestion extends React.Component {
 	recordRender() {
 		if ( this.props.railcarId && isNumber( this.props.uiPosition ) ) {
 			let resultSuffix = '';
-			if ( this.props.suggestion.isRecommended ) {
+			if ( this.props.suggestion.is_premium ) {
```

Comment:
@hambai, this construction will mask cases where the premium domain appears in position 1 or 2 (aka when the domain is recommended or the best alternative).

I think it would be better to leave the existing `if/else if` logic and add another `if` block to capture this.

Reply:
huh yeah - I've asked our data engineering team for help as to where/how to store that info

Reply:
After discussing this internally - p1601461479028900-slack-C03DKP1JP - we're going with additional suffix (not replacing the current suffix)